core/container/externalbuilder/copy.go

```diff
@@ -78,6 +88,14 @@ func CopyDir(srcroot, destroot string) error {
 	})
 }
 
+func copySymlink(srcpath, destpath string) error {
+	symlinkDest, err := os.Readlink(srcpath)
+	if err != nil {
+		return err
+	}
+	return os.Symlink(symlinkDest, destpath)
```

Comment:
The links must not point outside of the tree and must be relative. This does not appear to handle that.